src/App.js

```diff
@@ -4,12 +4,14 @@ import PureComponent from "./Task1/PureComponent";
 import FunctionComponentCounter from "./Task1/FunctionComponentAndCounter";
 import SearchForm from "./Task1/SearchForm";
 import SelectGenre from "./Task1/SelectGenre";
-import reactCreateElement from "./Task1/React.CreateElement";
+import CreateElement from "./Task1/React.CreateElement";
+import MainContainer from "./Task3/MainContainer";
```

Comment:
`MainContainer` doesn't exist in the repo. Seems like you were working on some other components upfront. Please, remove everything which is not related on this task from this branch and PR. Otherwise, it gets harder to focus on what I need to review. If you need to work on some other code upfront, please do it in other branches.

Reply:
I removed

Reply:
You removed `MainContainer` only. Please remove all other things unrelated to task 1 too please. E.g. images.

Reply:
I fixed